addons/pos_restaurant/static/src/app/floor_screen/floor_screen.js

```diff
@@ -56,15 +55,15 @@ export class FloorScreen extends Component {
     }
     onPatched() {
         this.floorMapRef.el.style.background = this.state.floorBackground;
-        if (!this.pos.isEditMode && this.pos.floors.length > 0) {
+        if (!this.pos.isEditMode && this.pos.models["restaurant.floor"].length > 0) {
```

Comment:
```suggestion
        if (!this.pos.isEditMode && this.pos.config.floor_ids.length > 0) {
```

Edit other instances.

Reply:
Like said bellow, we can keep it like that, no floor and table will be loaded if we're not in resto mode